tests/acceptance/helpers/webdavHelper.js

```diff
@@ -232,3 +232,30 @@ exports.uploadFileWithContent = function (user, content, filename) {
   })
     .then(res => httpHelper.checkStatus(res, 'Could not upload file' + filename + 'with content' + content))
 }
+
+exports.getFavouritedResources = function (user) {
+  const headers = httpHelper.createAuthHeader(user)
+  const body = `<oc:filter-files  xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns"><d:prop><d:resourcetype /></d:prop>
+         <oc:filter-rules>
+        <oc:favorite>1</oc:favorite>
+    </oc:filter-rules>
+</oc:filter-files>`
+  return fetch(client.globals.backend_url + `/remote.php/dav/files/${user}`,
+    {
+      method: 'REPORT',
+      headers,
+      body
+    })
+    .then(res => res.text())
+    .then(res => {
+      const favData = convert.xml2js(res, { compact: true })['d:multistatus']['d:response']
+      const favItems = []
+      favData.map(favourite => {
+        favItems.push({
+          href: favourite['d:href']._text,
```

Comment:
Do we really need the href here. Can't we just parse out the name of file or folder.

Reply:
fixed